presidio-analyzer/presidio_analyzer/nlp_engine/nlp_engine.py

```diff
@@ -53,3 +53,8 @@ def is_punct(self, word: str, language: str) -> bool:
     def get_supported_entities(self) -> List[str]:
         """Return the supported entities for this NLP engine."""
         pass
+
+    @abstractmethod
+    def get_supported_languages(self) -> List[str]:
```

Comment:
Added a `get_supported_languages` to NlpEngine, so that we create the same nlp recognizers as defined by the nlp engine, instead of taking the languages from the list of recognizers.